build.gradle

```diff
@@ -78,10 +78,14 @@ subprojects {
         funTestCompile project(':utils-test')
     }
 
-    // TODO: Remove this once jackson-module-kotlin and kotlintest are updated.
     configurations.all {
         resolutionStrategy {
+            // TODO: Remove this once jackson-module-kotlin and kotlintest are updated.
             force 'org.jetbrains.kotlin:kotlin-reflect:1.2.51'
+
+            // Required because Konf dependens on version 1.21 which is incompatible with Jackson 1.9.x:
```

Comment:
Isn't this supposed to say "Jackson 2.9.x"?

Reply:
Done.